en/docs/install-and-setup/setup/mi-setup/observability/setting-up-minimum-basic-observability-deployment.md

```diff
@@ -86,7 +86,7 @@ These dashboards are provided as JSON files that can be manually imported to Gra
 
     The **Import** dialog box opens as follows.
     
-    ![Import Dashboards dialog box](../../assets/img/monitoring-dashboard/grafana-import-dialog-box.png)
+    ![Import Dashboards dialog box]({{base_path}}/assets/img//integrate/monitoring-dashboard/grafana-import-dialog-box.png)
```

Comment:
{{base_path}}/assets/img//integrate/ should be {{base_path}}/assets/img/integrate/
Please remove the extra / in the path. This issue is there in other places as well.